feat: add toggle workflow description (#377)
Azure/Azure-Verified-Modules: docs/content/contributing/bicep/bicep-contribution-flow/enable-or-disable-workflows.md

Reviewer: Can we have a couple of screenshots to show the usage? E.g., add examples to disable all avm workflows except one, which will be the most common use case for module owners. 

I'd also suggest to leverage hugo language, adding notes and important notes to highlight, e.g., that the workflow only works in forks

Author: Done. Please review again

---

### _Navigation_

- [Location](#location)
- [How it works](#how-it-works)
- [Typical use cases](#typical-use-cases)
- [Limitations](#limitations)

---
# Location

You can find the script under [`avm/utilities/pipelines/platform/Switch-WorkflowSate.ps1)`](https://github.com/Azure/bicep-registry-modules/blob/main/avm/utilities/pipelines/platform/Switch-WorkflowSate.ps1)
You can find the workflow under [`.github/workflows/avm.platform.toggle-avm-workflows.yml`](https://github.com/Azure/bicep-registry-modules/blob/main/.github/workflows/avm.platform.toggle-avm-workflows.yml)

# How it works

Select the workflow from the list

<img src="../../img/contribution/selectToggleWorkflows.png" width=100%>

Run the workflow `avm.platform.toggle-avm-workflows` and set the following settings:

> - `targetState` to enable or disable workflows

Reviewer: ```suggestion
- `Enable or disable workflows` to enable or disable workflows
```

> - `includePattern` set the RegEx which workflows are included

Reviewer: ```suggestion
- `RegEx which workflows are included` include a specific set of workflows, using a RegEx.
```

> - `excludePattern` set the RegEx which workflows are excluded

Reviewer: ```suggestion
-  `RegEx which workflows are excluded` exclude a specific set of workflows, using a RegEx.
```

<img src="../../img/contribution/runToggleWorkflows.png" width=100%>

# Typical use cases

If all but one workflow should be disabled, set the following settings:

> - `targetState` to `disable`

Reviewer: ```suggestion
- `Enable or disable workflows` to `Disable`
```

> - `includePattern` to `avm\.(?:res|ptn)` (that is the default setting)

Reviewer: ```suggestion
-  `RegEx which workflows are included` to `avm\.(?:res|ptn)` (that is the default setting)
```

> - `includePattern` to `avm.res.compute.virtual-machine` (the name of your workflow, here as example the one for the virtual machine)

Reviewer: ```suggestion
- `RegEx which workflows are excluded` to `avm.res.compute.virtual-machine` (the name of your workflow, here as example the one for the virtual machine)
```

If all workflows should be enabled, set the following settings:

> - `targetState` to `enable`

Reviewer: ```suggestion
- `Enable or disable workflows` to `Enable`
```

> - `includePattern` to `avm\.(?:res|ptn)` (that is the default setting)

Reviewer: ```suggestion
- `RegEx which workflows are included` to `avm\.(?:res|ptn)` (that is the default setting)
```

> - `includePattern` to `^$` (that is the default setting)

Reviewer: ```suggestion
- `RegEx which workflows are excluded` to `^$` (that is the default setting)
```
